### ISSUE TITLE
Cards (example) not working

### ISSUE THREAD

**Issue by DJF3** [CiscoDevNet/webexteamssdk#155]
The cards functionality doesn't seem to work. If that is confirmed, can we remove the code and the documentation? 

```
from webexteamssdk.cards.card import AdaptiveCard
from webexteamssdk.cards.components import TextBlock
from webexteamssdk.cards.options import Colors, HorizontalAlignment
from webexteamssdk.cards.inputs import Text, Number
from webexteamssdk.cards.actions import Submit
```
Always results in: "ModuleNotFoundError: No module named 'webexteamssdk.cards'"

Can someone confirm that the cards functionality is broken (and won't be fixed)?

**Comment by marksull** [comment]
@DJF3 Insert ".model." before ".cards". For example:

```
```

**Comment by DJF3** [comment]
When I apply these changes to the example shown [here](https://webexteamssdk.readthedocs.io/en/latest/user/cards.html)   this error appears:

`TypeError: Object of type AdaptiveCard is not JSON serializable`

**Comment by marksull** [comment]
Yes, that is caused by a bug that will be resolved by this pull request: https://github.com/CiscoDevNet/webexteamssdk/pull/141

Either wait for the pull request to be merged, or use the above branch until then, or instead of passing in the ActiveCard, pass in the raw dictionary for the attachement:

For example, instead of using this:
```
api.messages.create(text="fallback", roomId="...", cards=card)
```

replace it with: 
```
from webexteamssdk.utils import make_attachment
...
api.messages.create(text="fallback", roomId="...", attachments=[make_attachment(card)])
```

That should provide a good workaround for now.

**Comment by DJF3** [comment]
Great, it's working now.

Thanks for the quick support!

**Comment by DJF3** [comment]
For those who need a working example with more features used: https://github.com/DJF3/Webex-SDK-Cards-example
